src/routes/launchpad/+page.svelte

```diff
@@ -0,0 +1,676 @@
+						Toast.success(`Successful transfer at block ${(result as { Ok: bigint }).Ok}`)
+					);
+			}
+		} catch (e) {
+			console.error(e);
+		}
+		isNotAvailable = false;
+	};
+
+	const updateBalance = async () => {
+		if (!destination || !$canisters) return;
+		try {
+			const { e8s } = await $canisters.icpLedger.anonymousActor.account_balance({
+				account: AccountIdentifier.fromHex(destination).toUint8Array()
+			});
+			balance = e8s;
+		} catch (e) {
+			console.log(e);
+		}
+	};
+
+	const fetchStatus = async () => {
+		try {
+			status = await snsCanister.get_status();
+			resetTimerToElapsedTime(status.time_left / 1_000_000_000n);
+			startTimer();
+		} catch (e) {
+			console.log(e);
+		}
+	};
+
+	const setDestination = async () => {
+		isNotAvailable = true;
+		try {
+			destination = await snsCanister.get_icp_deposit_address(Principal.fromText(participant));
+			await updateBalance();
+			await updateWtnAllocation();
+		} catch (e) {
+			console.log(e);
+		}
+		isNotAvailable = false;
+	};
+
+	const notifyDeposit = async () => {
+		isNotAvailable = true;
+		try {
+			const result = await snsCanister.notify_icp_deposit(Principal.fromText(participant), balance);
+			const key = Object.keys(result)[0];
+			switch (key) {
+				case 'Err':
+					toasts.add(Toast.error((result as { Err: string }).Err));
+					break;
+				case 'Ok':
+					toasts.add(
+						Toast.success(`Successful transfer at block ${(result as { Ok: bigint }).Ok}`)
+					);
+			}
+		} catch (e) {
+			console.log(e);
+		}
+		isNotAvailable = false;
+	};
+
+	onMount(() => {
+		setUpSns().then(async () => {
+			fetchStatus();
+			setPositions();
+		});
+
+		const intervalId = setInterval(async () => {
+			await fetchStatus();
+			await updateBalance();
+			await updateWtnAllocation();
+			setPositions();
+		}, 5000);
+
+		return () => clearInterval(intervalId);
+	});
+
+	$: status;
+</script>
+
+<main>
+	<div class="header-container">
+		<p>
+			WaterNeuron is a liquid staking protocol designed for the Internet Computer network. Staking
+			ICP becomes straightforward and efficient.
+		</p>
+		<span>https://docs.waterneuron.fi | https://x.com/waterneuron | https://t.me/waterneuron</span>
+	</div>
+	<div class="core-container">
+		<div class="parameters-container">
+			<div class="parameter">
+				<span>Token Name</span> <span>WaterNeuron</span>
+			</div>
+
+			<div class="parameter">
+				<span>Token Symbol</span> <span>WTN</span>
+			</div>
+
+			<div class="parameter">
+				<span>Total Supply</span> <span>116'479'684.72 WTN</span>
+			</div>
+			<div class="parameter">
+				<span>Minimum Participants</span> <span>Unlimited</span>
+			</div>
+
+			<div class="parameter">
+				<span>Minimum Participant Commitment</span> <span>10 ICP</span>
+			</div>
+
+			<div class="parameter">
+				<span>Maximum Participant Commitment</span> <span>Unlimited</span>
+			</div>
+
+			<div class="parameter">
+				<span>Swap End</span> <span>Jul 3, 2024 6:00 PM</span>
+			</div>
```

Comment:
You can add swap start, and dynamically show the real date not a mock one.

Reply:
yes, done